src/crypto/sha256.cpp

```diff
@@ -484,6 +484,14 @@ void inline cpuid(uint32_t leaf, uint32_t subleaf, uint32_t& a, uint32_t& b, uin
 {
   __asm__ ("cpuid" : "=a"(a), "=b"(b), "=c"(c), "=d"(d) : "0"(leaf), "2"(subleaf));
 }
+
+/** Check whether the OS has enabled AVX registers. */
+bool AVXEnabled()
+{
+    uint32_t a, d;
```

Comment:
Does this need ```#ifdef USE_ASM``` ? I'm really not even sure what the option is intended to mean anymore.

Reply:
Indeed, nevermind.